### ISSUE TITLE
1.20.1 does not work on server

### ISSUE THREAD

**Issue by HateAndLoveModding** [iChun/MultiplayerServerPause#14]
I just wanted to let you know that the 1.20.1 file does not work on the server. It works on the client, but you need it on at least the server for it to work. I downloaded the code and looked into the issue. It seems that it is an issue with completeconfig. I put the actual true false values in the java code, deleted all the config, and rebuilt the jar and now it works great. I have no idea if you want to do anything about this issue, but I thought I would let you know anyway. Btw, I am pretty sure that all other versions work just file. I did test all the versions of completeconfig but none of them worked.

**Comment by HateAndLoveModding** [comment]
Oh did I mention that I am using Fabric?

**Comment by Partonetrain** [comment]
Crash report:
```
[21:32:37] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'serverpause'!
        at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:63) ~[fabric-loader-0.15.11.jar:?]
        at net.minecraft.server.Main.main(Main.java:112) ~[server-intermediary.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) ~[fabric-loader-0.15.11.jar:?]
        at net.fabricmc.installer.ServerLauncher.main(ServerLauncher.java:69) ~[fabric-server-mc.1.20.1-loader.0.15.11-launcher.1.0.1.jar:1.0.1]
Caused by: java.lang.NoSuchFieldError: checkbox
        at me.lortseam.completeconfig.data.BooleanEntry.<init>(BooleanEntry.java:25) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Entry.create(Entry.java:42) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.EntrySet.lambda$resolve$1(EntrySet.java:31) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?]
        at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
        at me.lortseam.completeconfig.data.EntrySet.resolve(EntrySet.java:32) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Parent.resolveContainer(Parent.java:58) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.ClusterSet.resolve(ClusterSet.java:13) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Parent.resolve(Parent.java:108) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Parent.lambda$resolveContainer$6(Parent.java:97) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?]
        at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:992) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
        at me.lortseam.completeconfig.data.Parent.resolveContainer(Parent.java:97) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Parent.resolve(Parent.java:110) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Config.lambda$new$0(Config.java:52) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Config.deserialize(Config.java:94) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.lortseam.completeconfig.data.Config.load(Config.java:121) ~[completeconfig-base-2.5.0-cd5950f26b406f27.jar:?]
        at me.ichun.mods.serverpause.loader.fabric.LoaderFabric.onInitialize(LoaderFabric.java:24) ~[MultiplayerServerPause-1.20.1-Fabric-1.1.0.jar:?]
        at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384) ~[fabric-loader-0.15.11.jar:?]
        ... 7 more
```

**Comment by iChun** [comment]
Ah you need to use Fabric Loader 0.14.x for 1.20.1 as 0.15+ breaks complete config. It's fixed for later versions but I don't think this mod is updated anymore and it's unlikely to be fixed for 1.20.1 https://github.com/Lortseam/completeconfig/issues/26#issuecomment-2009262429
